packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -269,8 +268,8 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
         _lastPendingEventTimestamp = event.timeStamp;
         _lastTransform = event.transform;
         final Offset movedLocally = _getDeltaForDetails(event.localDelta);
-        final Matrix4 localToGlobalTransform = event.transform == null ? null : Matrix4.tryInvert(event.transform);
-        _globalDistanceMoved += PointerEvent.transformDeltaViaPositions(
+        final Matrix4? localToGlobalTransform = event.transform == null ? null : Matrix4.tryInvert(event.transform!);
+        _globalDistanceMoved = _globalDistanceMoved + PointerEvent.transformDeltaViaPositions(
```

Comment:
This one can go back to `+=` as well then?

Reply:
Fixed. Thanks!